chat-plugins/info.js

```diff
@@ -183,7 +183,7 @@ exports.commands = {
 			return this.errorReply("/checkpunishment - Access denied.");
 		}
 		let userid = toId(target);
-		let buf = Chat.html`<strong class="username">${userid}</strong> <em style="color:gray">(offline)</em><br /><br />`;
+		let buf = Chat.html`<strong class="username">${Chat.escapeHTML(target)}</strong> <em style="color:gray">(offline)</em><br /><br />`;
```

Comment:
Actually, because of ``Chat.html``, I don't need to do ``Chat.escapeHTML(target)`` here, I believe